src/main/java/ch/njol/skript/expressions/ExprBlockPower.java

```diff
@@ -0,0 +1,46 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.block.Block;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+
```

Comment:
You are missing docs here. This should include the name (name of the expression), description (describe what this expression does), examples (a few examples showing how this works) and version ("INSERT VERSION" <-- that should be used, we replace this later during release)